test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,30 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
+	NodeAffinity nodeAffinity
+	InterpodAffinity interpodAffinity
+}
+
+// InterpodAffinity priority config input values that can be fed to scheduler.
```

Comment:
`InterpodAffinity priority configuration details.` Is all thats needed for a comment here.